src/packaging/docker-services/generate-ssl-stores/docker-entrypoint.sh

```diff
@@ -73,24 +80,33 @@ then
         esac
     done
 
-fi
+ fi
 set -x
 
 for store_name in ${CASSANDRA_KEYSTORE} ${REAPER_KEYSTORE} ${GENERIC_TRUSTSTORE}
 do
     rm -f ${store_name}
 done
 
+if [ -f "$CA_KEY" ] ; then
+    echo "$CA_KEY" exists, will not create
+else
+   openssl genrsa -out ${CA_KEY} 2048
+fi
+
+if [ -f "$ROOT_CA_CERT" ] ; then
+    echo "$ROOT_CA_CERT" exists, will not create
+else
+# Create the Root Certificate Authority (Root CA) from the Certificate Authority Configuration and verify contents.
+   openssl req -x509 -sha256 -new -nodes -key ca-key -out ${ROOT_CA_CERT} -config ${CA_CERT_CONFIG}
+fi
+
 set +x
 echo
 echo "Generic Certificate Authority configuration"
 cat ${CA_CERT_CONFIG}
-echo
 set -x
 
-# Create the Root Certificate Authority (Root CA) from the Certificate Authority Configuration and verify contents.
```

Comment:
Leave this line in OR if we are generating the `ca-key` as mentioned above, perhaps add that to the comment.